addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -1737,7 +1737,7 @@ var UrlWidget = InputField.extend({
         }
         this.el.classList.add("o_form_uri", "o_text_overflow");
         const anchorEl = Object.assign(document.createElement('a'), {
-            text: this.attrs.text || this.value,
+            text: _t(this.attrs.text) || this.value,
```

Comment:
The content of `this.attrs.text` will not be translated, the `_t` function needs an explicit string.
I think we have to translate it in backend by adding the `text` attribute in the `TRANSLATED_ATTRS` variable defined in this [file](https://github.com/odoo/odoo/blob/5ef48dbdfff8fbaf6dd6f3985299271ef0ceaf34/odoo/tools/translate.py#L149-L156) like this :
```diff
--- a/odoo/tools/translate.py
+++ b/odoo/tools/translate.py
@@ -151,9 +151,12 @@ TRANSLATED_ATTRS = dict.fromkeys({
     'aria-keyshortcuts', 'aria-placeholder', 'aria-roledescription', 'aria-valuetext',
     'value_label',
 }, lambda e: True)
-TRANSLATED_ATTRS['value'] = lambda e: (e.tag == 'input' and e.attrib.get('type', 'text') == 'text') and 'datetimepicker-input' not in e.attrib['class'].split(' ')
-
-TRANSLATED_ATTRS.update({f't-attf-{attr}': cond for attr, cond in TRANSLATED_ATTRS.items()})
+TRANSLATED_ATTRS.update(
+    value=lambda e: (e.tag == 'input' and e.attrib.get('type', 'text') == 'text') and 'datetimepicker-input' not in e.attrib['class'].split(' '),
+    # xbo => text attribute to translate when the node is field tag and this node contains widget='url' in its attributes.
+    text=lambda e: (e.tag == 'field' and e.attrib.get('widget', '') == 'url'),
+    **{f't-attf-{attr}': cond for attr, cond in TRANSLATED_ATTRS.items()},
+)
 
 avoid_pattern = re.compile(r"\s*<!DOCTYPE", re.IGNORECASE | re.MULTILINE | re.UNICODE)
 node_pattern = re.compile(r"<[^>]*>(.*)</[^<]*>", re.DOTALL | re.MULTILINE | re.UNICODE)
```

@mart-e What do you think ? 🙂 

cc @IT-Ideas

Reply:
Yes, should do the trick :+1: